homeassistant/components/philips_js/__init__.py

```diff
@@ -64,11 +64,12 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry):
 class PluggableAction:
     """A pluggable action handler."""
 
-    _actions: Dict[Any, AutomationActionType] = {}
+    _actions: Dict[Any, AutomationActionType]
```

Comment:
I suggest moving the typing to the init method. Having it here is just another place that you need to look up.

Reply:
fixed in #45784